src/services/provider/routingCategories.provider.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @module services/provider/routingCategories_provider
+ */
+import { $injector } from '../../injection/index';
+
+/**
+ * Bvv specific implementation of {@link module:services/RoutingService~routingCategoriesProvider}
+ * @function
+ * @type {module:services/FeedbackService~feedbackStorageProvider}
```

Comment:
doc typo
should be `* @type {module:services/RoutingService~routingCategoriesProvider}`